pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -768,6 +767,15 @@ func TestInsufficientCapacityNodeSufficientCapacityWithNodeLabelDaemonLaunchPod(
 }
 
 func TestSufficientCapacityWithTerminatedPodsDaemonLaunchesPod(t *testing.T) {
+	enabled := utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods)
+	// Rollback feature gate.
+	defer func() {
+		if enabled {
+			utilfeature.DefaultFeatureGate.Set("ScheduleDaemonSetPods=true")
+		}
+	}()
+
+	utilfeature.DefaultFeatureGate.Set("ScheduleDaemonSetPods=false")
```

Comment:
As above, why is the feature disabled in this test?

Reply:
We would launch(create) the pods, even with insufficient capacity on the nodeafter this feature has been enabled, it is the job of the scheduler to check for resource usage on the node.